lua/nvim-web-devicons/icons-default.lua

```diff
@@ -2327,18 +3149,42 @@ local icons_by_operating_system = {
     cterm_color = "239",
     name = "Solus",
   },
+  ["tails"] = {
+    icon = "",
+    color = "#56347c",
+    cterm_color = "54",
+    name = "Tails",
+  },
+  ["trisquel"] = {
+    icon = "",
+    color = "#0f58b6",
+    cterm_color = "25",
+    name = "Trisquel_GNU-Linux",
+  },
   ["ubuntu"] = {
     icon = "",
     color = "#dd4814",
     cterm_color = "196",
     name = "Ubuntu",
   },
+  ["vanillaos"] = {
+    icon = "",
+    color = "#fabd4d",
+    cterm_color = "214",
+    name = "VanillaOS",
+  },
   ["void"] = {
     icon = "",
     color = "#295340",
     cterm_color = "23",
     name = "Void",
   },
+  ["xerolinux"] = {
+    icon = "",
```

Comment:
Should be `nf-linux-xerolinux`

Reply:
already is

Reply:
Sorry, must have failed to copy the correct icon to the cheat sheet.